Mathlib/Data/Rat/Cast/Order.lean

```diff
@@ -132,6 +132,10 @@ def castOrderEmbedding : ℚ≥0 ↪o K :=
 @[simp] lemma cast_pos : (0 : K) < q ↔ 0 < q := by norm_cast
 @[norm_cast] lemma cast_lt_zero : (q : K) < 0 ↔ q < 0 := by norm_cast
 @[simp] lemma not_cast_lt_zero : ¬(q : K) < 0 := mod_cast not_lt_zero'
+@[simp, norm_cast] lemma cast_le_one : (p : K) ≤ 1 ↔ p ≤ 1 := by norm_cast
```

Comment:
Given that `norm_cast` can already prove it, does this need to be tagged as such?

Reply:
Maybe not... I guess we don't have `normCastNF`